src/examples/CMakeLists.txt

```diff
@@ -1,11 +1,16 @@
 # SPDX-License-Identifier: BSD-3-Clause
 # Copyright (c) Contributors to the OpenEXR Project.
 
-cmake_minimum_required(VERSION 3.12)
+# If the project name is set, this is being configured as a # part of
```

Comment:
Is the hash sign before 'part' a leftover from a moved line break or intentional?

Reply:
Good catch, thanks. Yes, a reformatting snafu.